1.8.0/director/templates/dep-maya-io.yaml

```diff
@@ -0,0 +1,177 @@
+        - "-c"
+        - |
+          set -ex
+          export COUNT=0
+          echo $COUNT
+          until [ $COUNT -ne 0 ]
+          do
+             echo "wating for maya-grafana"
+             sleep 2;
+             export COUNT=$(kubectl get pods -l app=maya-grafana -n {{ .Release.Namespace }} -o json  | jq -r '.items[] | select(.status.phase == "Running" and ([ .status.conditions[] | select(.type == "Ready" and .status == "True") ] | length ) >= 1 ) | .metadata.name' | grep -w maya-grafana | wc -l)
+             echo $COUNT
+          done
+      - name: mios-wait-for-maya-ui
+        image: registry.mayadata.io/maya-init:1.0.0
+        command:
+        - sh
+        - "-c"
+        - |
+          set -ex
+          export COUNT=0
+          echo $COUNT
+          until [ $COUNT -ne 0 ]
+          do
+             echo "wating for maya-ui"
+             sleep 2;
+             export COUNT=$(kubectl get pods -l app=maya-ui -n {{ .Release.Namespace }} -o json  | jq -r '.items[] | select(.status.phase == "Running" and ([ .status.conditions[] | select(.type == "Ready" and .status == "True") ] | length ) >= 1 ) | .metadata.name' | grep -w maya-ui | wc -l)
+             echo $COUNT
+          done
+      - name: mios-wait-for-elasticsearch-logging
+        image: registry.mayadata.io/maya-init:1.0.0
+        command:
+        - sh
+        - "-c"
+        - |
+          set -ex
+          export COUNT=0
+          echo $COUNT
+          until [ $COUNT -ne 0 ]
+          do
+             echo "wating for elasticsearch-logging"
+             sleep 2;
+             export COUNT=$(kubectl get pods -l k8s-app=od-elasticsearch-logging -n {{ .Release.Namespace }} -o json  | jq -r '.items[] | select(.status.phase == "Running" and ([ .status.conditions[] | select(.type == "Ready" and .status == "True") ] | length ) >= 1 ) | .metadata.name' | grep -w od-elasticsearch-logging | wc -l)
+             echo $COUNT
+          done
+      - name: mios-wait-for-kibana-logging
+        image: registry.mayadata.io/maya-init:1.0.0
+        command:
+        - sh
+        - "-c"
+        - |
+          set -ex
+          export COUNT=0
+          echo $COUNT
+          until [ $COUNT -ne 0 ]
+          do
+             echo "wating for kibana-logging"
+             sleep 2;
+             export COUNT=$(kubectl get pods -l k8s-app=od-kibana-logging -n {{ .Release.Namespace }} -o json  | jq -r '.items[] | select(.status.phase == "Running" and ([ .status.conditions[] | select(.type == "Ready" and .status == "True") ] | length ) >= 1 ) | .metadata.name' | grep -w od-kibana-logging | wc -l)
+             echo $COUNT
+          done
+      - name: mios-wait-for-alertstore-tablemanager
+        image: registry.mayadata.io/maya-init:1.0.0
+        command:
+        - sh
+        - "-c"
+        - |
+          set -ex
+          export COUNT=0
+          echo $COUNT
+          until [ $COUNT -ne 0 ]
+          do
+             echo "wating for alertstore-tablamanager"
```

Comment:
wating -> waiting , alertstore-tablamanager -> alertstore-tablemanager (in all previous releases this typo is present)